man/ncbi_snp_query_api.Rd

```diff
@@ -0,0 +1,88 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/ncbi_snp_api.R
+\name{ncbi_snp_query_api}
+\alias{ncbi_snp_query_api}
+\title{Query NCBI's refSNP for information on a set of SNPs via the API}
+\usage{
+ncbi_snp_query_api(SNPs, ...)
+}
+\arguments{
+\item{SNPs}{(character) A vector of SNPs (rs numbers).}
+
+\item{...}{Curl options passed on to \link[crul:HttpClient]{crul::HttpClient}}
+}
+\value{
+A dataframe with columns:
+\itemize{
+\item Query: The rs ID that was queried.
+\item Chromosome: The chromosome that the marker lies on.
+\item BP: The chromosomal position, in base pairs, of the marker,
+as aligned with the current genome used by dbSNP. we add 1 to the base
+pair position in the BP column in the output data.frame to agree with
+what the dbSNP website has.
+\item Marker: The name of the marker. If the rs ID queried
```

Comment:
Have you thought about renaming marker with rsid? or snp? I think "marker" is a bit outdated.